src/node.cpp

```diff
@@ -1496,7 +1496,7 @@ void LocalNode::setnotseen(int newnotseen)
 }
 
 // set fsid - assume that an existing assignment of the same fsid is no longer current and revoke
-void LocalNode::setfsid(handle newfsid)
+void LocalNode::setfsid(handle newfsid, handlelocalnode_map& fsidnodes)
```

Comment:
clearer if we access sync->client->fsidnode directly, rather than passing it in by parameter - no need for this change

Reply:
See my above above for the header.